packages/gatsby-source-wordpress/src/normalize.js

```diff
@@ -138,7 +138,10 @@ exports.createGatsbyIds = entities =>
       namespace = typeNamespaces[e.__type]
     }
 
-    e.id = uuidv5(e.wordpress_id.toString(), namespace)
+    e.id = uuidv5(
+      e.wordpress_id ? e.wordpress_id.toString() : Math.floor((1 + Math.random()) * 0x10000).toString(16),
```

Comment:
This won't work because IDs need to be consistent across builds or otherwise the cache won't work.

I suspect that we can safely ignore entities from the wordpress api without ids as they're not data we care about per my comment in the issue.

Reply:
Wordpress endpoints universe isn't normalized, there is surely somewhere some plugins that return data without any ID.

Would this be a solution ?

```javascript
 e.id = uuidv5(
      e.wordpress_id ? e.wordpress_id.toString() : stringify(e),
      namespace
    )
    return e
```